src/Generator.php

```diff
@@ -141,7 +141,10 @@ private function getRoutes(): Collection
                 $routeResolver = Scramble::$routeResolver ?? function (Route $route) {
                     $expectedDomain = config('scramble.api_domain');
 
-                    return Str::startsWith($route->uri, config('scramble.api_path', 'api'))
+                    $routePrefix = trim(config('scramble.api_path', 'api'), '/');
+                    $routeRegex = '/^' . preg_quote($routePrefix, '/') . '((\/.+)|$)/';
+
+                    return preg_match($routeRegex, $route->uri)
```

Comment:
Does this regexp equal to this approach? If so, I'd prefer `startsWith`.

```suggestion
                    return Str::startsWith($route->uri, rtrim(config('scramble.api_path', 'api'), '/').'/');
```

Reply:
I think it should be `Str::startsWith($route->uri.'/', rtrim(config('scramble.api_path', 'api'), '/').'/')`: with that fix yeah, it actually does the very same thing. Gonna change that.